benchmarks/package.json

```diff
@@ -0,0 +1,3 @@
+{
+  "type": "module"
```

Comment:
Temporary workaround since the top-level `package.json` does not use pure ES modules.
To remove once this is not the case anymore.